apps.toml

```diff
@@ -1,5 +1,3 @@
-#:schema https://github.com/YunoHost/apps/raw/master/schemas/apps.toml.schema.json
-
```

Comment:
Hmmmm, uhoh ?

Reply:
oh no

Reply:
Ah yes we might want to use tomlkit instead of toml (if that's not already the case?)

Reply:
Meh not better with tomlkit…

Reply:
See https://github.com/YunoHost/apps/pull/2138